DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -0,0 +1,1024 @@
+      bookME(ibooker,DeltaR_jet_Mu_,histname,histtitle, DR_binning_.nbins, DR_binning_.xmin, DR_binning_.xmax );      
+      setMETitle(DeltaR_jet_Mu_,"#DeltaR(jet,mu)","events");
+  }
+
+  histname = "htVsLS"; histtitle = "event HT vs LS";
+  bookME(ibooker,htVsLS_,histname,histtitle,ls_binning_.nbins, ls_binning_.xmin, ls_binning_.xmax,pt_binning_.xmin, pt_binning_.xmax);
+  setMETitle(htVsLS_,"LS","event HT [GeV]");
+
+  histname = "metPhi"; histtitle = "PFMET phi";
+  bookME(ibooker,metPhiME_,histname,histtitle, phi_binning_.nbins, phi_binning_.xmin, phi_binning_.xmax);
+  setMETitle(metPhiME_,"PF MET #phi","events / 0.1 rad");
+
+  if ((muPt_.size()!=nmuons_) || (muEta_.size()!=nmuons_) || (muPhi_.size()!=nmuons_)){
+      edm::LogWarning("TopMonitor") << "Number of histograms does not match with number of required muons \n";
+      return;
+  } 
+
+  for (unsigned int iMu=0; iMu<nmuons_; ++iMu){
+      std::string index = std::to_string(iMu+1);
+
+      histname = "muPt_"; histtitle = "muon p_{T} - ";
+      histname.append(index); histtitle.append(index);
+      bookME(ibooker,muPt_.at(iMu),histname,histtitle, pt_binning_.nbins, pt_binning_.xmin, pt_binning_.xmax);
+      setMETitle(muPt_.at(iMu),"muon p_{T} [GeV]","events");
+      histname.append("_variableBinning");
+      bookME(ibooker,muPt_variableBinning_.at(iMu),histname,histtitle, muPt_variable_binning_);
+      setMETitle(muPt_variableBinning_.at(iMu),"muon p_{T} [GeV]","events");
+
+      histname = "muEta_"; histtitle = "muon #eta - ";
+      histname.append(index); histtitle.append(index);
+      bookME(ibooker,muEta_.at(iMu),histname,histtitle, eta_binning_.nbins,eta_binning_.xmin, eta_binning_.xmax);
+      setMETitle(muEta_.at(iMu)," muon #eta","events");
+      histname.append("_variableBinning");
+      bookME(ibooker,muEta_variableBinning_.at(iMu),histname,histtitle, muEta_variable_binning_);
+      setMETitle(muEta_variableBinning_.at(iMu)," muon #eta","events");
+
+      histname = "muPhi_"; histtitle = "muon #phi - ";
+      histname.append(index); histtitle.append(index);
+      bookME(ibooker,muPhi_.at(iMu),histname,histtitle, phi_binning_.nbins, phi_binning_.xmin, phi_binning_.xmax);
+      setMETitle(muPhi_.at(iMu)," muon #phi","events");
+
+      histname = "muPtEta_"; histtitle = "muon p_{T} - #eta - ";
+      histname.append(index); histtitle.append(index);
+      bookME(ibooker,muPtEta_.at(iMu),histname,histtitle, muPt_variable_binning_2D_, muEta_variable_binning_2D_);
+      setMETitle(muPtEta_.at(iMu),"muon p_{T} [GeV]","muon #eta");
+
+      histname = "muEtaPhi_"; histtitle = "muon #eta - #phi - ";
+      histname.append(index); histtitle.append(index);
+      bookME(ibooker,muEtaPhi_.at(iMu),histname,histtitle, muEta_variable_binning_2D_, phi_variable_binning_2D_);
+      setMETitle(muEtaPhi_.at(iMu),"muon p_{T} [GeV]","muon #eta");
+
+
+  }
+
+  if ((elePt_.size()!=nelectrons_) || (eleEta_.size()!=nelectrons_) || (elePhi_.size()!=nelectrons_)){
```

Comment:
yes, those lines were there only as sanity checks for debugging purposes, I forgot to remove them